scripts/ecs_schema.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+require('../src/setup_node_env/no_transpilation');
```

Comment:
Does using `no_transpilation` fix the issues we were looking working through?

Reply:
It did not.

I didn't realize I hadn't deleted this file yet. The only way I could get it to work was the way you discovered.